src/commponents/TodoItem.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import User from './User';
+
+function TodoItem(props) {
+
+
```

Comment:
remove the empty line.
please follow style guide - https://github.com/mate-academy/style-guides/blob/master/javascript.md#table-of-contents